### PR TITLE
Structure examples with appropriate `<main>`, `<header>`, `<footer>`

### PR DESCRIPTION
**Pull request by patrickhlauke**
Closes https://github.com/twbs/bootstrap/issues/23840

### CODE REVIEW

File: docs/4.0/examples/sticky-footer/index.html [hunk 1]
```diff
@@ -19,13 +19,13 @@
   <body>
 
     <!-- Begin page content -->
-    <div class="container">
+    <main role="container" class="container">
```
Comment (XhmikosR) [inline docs/4.0/examples/sticky-footer/index.html]: Typo here.

**Review by XhmikosR — CHANGES_REQUESTED** [review]

File: docs/4.0/examples/sticky-footer/index.html (hunk 1 above)
Comment (patrickhlauke) [reply docs/4.0/examples/sticky-footer/index.html]: Good spot!

Comment (XhmikosR) [reply docs/4.0/examples/sticky-footer/index.html]: vnu.jar in action, which is why we need to bring the HTML validator back. :)

Comment (patrickhlauke) [reply docs/4.0/examples/sticky-footer/index.html]: Fixed

**Review by XhmikosR — APPROVED** [review]

File: docs/4.0/examples/sticky-footer/index.html (hunk 1 above)
Comment (XhmikosR) [reply docs/4.0/examples/sticky-footer/index.html]: Don't forget to add the PR in the ship notes.

**Comment by vanillajonathan** [comment]
Consider also looking into the `nav` element.

**Comment by patrickhlauke** [comment]
most examples seem to have it, but i wasn't looking for that specifically. good call, will be a follow-up pr

**Pull request merged**